addon/components/frost-button.js

```diff
@@ -1,14 +1,11 @@
+/**
+ * Component definition for the frost-button component
+ */
 import Ember from 'ember'
-const {
-  Component,
-  get,
-  isEmpty,
-  Logger,
-  typeOf,
-  ViewUtils
-} = Ember
+const {Component, Logger, ViewUtils, get, isEmpty, typeOf} = Ember
```

Comment:
I have a  hunch that you can't actually use destructuring inside an `import`, based on [this](https://esdiscuss.org/topic/import-statements-destructuring) which makes sense, b/c the import syntax is static and evaluated at compile time, not run time. Similar to why you can't use a variable name in your `from`.